packages/contracts/test/AccessControlTest.js

```diff
@@ -336,14 +340,14 @@ contract('Access Control: Liquity functions with the caller restricted to Liquit
     // --- onlyActivePool ---
 
     // fallback (payment)	
-    it("fallback(): reverts when called by an account that is not the Active Pool", async () => {
+    it("fallback(): reverts because cannot receive ETH", async () => {
       // Attempt call from alice
       try {
         const txAlice = await web3.eth.sendTransaction({ from: alice, to: stabilityPool.address, value: 100 })
         
       } catch (err) {
         assert.include(err.message, "revert")
-        assert.include(err.message, "StabilityPool: Caller is not ActivePool")
+        //assert.include(err.message, "StabilityPool: Caller is not ActivePool")
```

Comment:
:thinking:

Reply:
The tx reverts without that msg because the StabilityPool contract cannot receive ETH anymore